src/main/app/src/utils/koulutus/getOsaamisalaLaajuus.ts

```diff
@@ -0,0 +1,42 @@
+import iterateTree from '#/src/utils/iterateTree';
+
+const getOsaamisalaLaajuus = (ePerusteOsat, osaamisalakoodi) => {
+  const osat = ePerusteOsat.map(ePerusteOsa => {
```

Comment:
Tähän voisi laittaa kommentin, että levitetään rekursiivisesti puussa alaspäin ePerusteen osien muodostumissäännöt, jotta voidaan näyttää ylätason laajuus niille osaamisaloille, joilta se puuttuu.

Reply:
Ja tosiaan suomeksi saa kommentoida.